combat-trainer.lic

```diff
@@ -948,11 +948,11 @@ class LootProcess
         dispose_trash(right_hand) if snap.last != right_hand && !@equipment_manager.is_listed_item?(right_hand)
       end
       game_state.need_bundle = false
-      unless stored_moon && bput('get moon', 'you grab', 'What were') == 'you grab'
+      unless stored_moon && DRCMM.hold_moon_weapon?
```

Comment:
Another candidate for a separate PR. Not necessary, though.

Reply:
Resolved in https://github.com/rpherbig/dr-scripts/pull/5396